e2e/terraform/modules/e2e/main.tf

```diff
@@ -72,7 +66,7 @@ resource "google_compute_instance" "e2e_instances" {
   }
   provisioner "file" {
     source      = "/etc/nephio/nephio.yaml"
-    destination = "/home/ubuntu/nephio.yaml"
+    destination = local.nephioyaml
```

Comment:
BTW, I think this file is only necessary for one summit repo, so we can remove it once we deprecated the usage of that repo.

Reply:
Yes, I agree we need remove the 'one-summit-demo' related stuff as part of a separate PR